_includes/footer.html

```diff
@@ -1,6 +1,6 @@
 <footer class="footer">
     <div class="content has-text-centered">
-        <p>Project Waterfall, © 2019 Michael Pyne.</p>
+        <p>Project Waterfall, © 2021 Michael Pyne.</p>
```

Comment:
I thought there was a change to year 2020?

Reply:
There is, in commit fd80fb8, but that was superseded by commit fa0cd59 in this pull request.

Should I squash these two commits into one overall commit and update the PR?

Reply:
No, but let's wait to resolve this nit until training tomorrow. :wink: